ovirt-web-ui.spec.in

```diff
@@ -20,19 +20,22 @@ Source0:        https://github.com/oVirt/ovirt-web-ui/archive/%{source_basename}
 
 ExclusiveArch:  x86_64
 
-BuildRequires: ovirt-engine-nodejs = 6.9.1
-# Keep ovirt-engine-nodejs-modules at particular version unless tested on higher
-BuildRequires: ovirt-engine-nodejs-modules = 0.0.17
+# Keep ovirt-engine-{nodejs|nodejs-modules|yarn} at particular version unless tested on higher
+# Please keep in sync with the automation/*.req files
+BuildRequires: ovirt-engine-nodejs=6.9.4
+BuildRequires: ovirt-engine-nodejs-modules=1.0.3
+BuildRequires: ovirt-engine-yarn=0.19.1
 
 %description
 This package provides new User Portal for %{product}, so far as technical preview.
 
 %prep
 # Use the ovirt-engine nodejs installation
-export PATH="%{_datadir}/ovirt-engine-nodejs/bin:${PATH}"
+#export PATH="%{_datadir}/ovirt-engine-nodejs/bin:${PATH}"
+source /usr/share/ovirt-engine-nodejs-modules/setup-env.sh
```

Comment:
This ^ lineed needs to go after the `%setup ...` line, as it modifyes the `yarn.lock` file, which is extracted by that `%setup ...` line.

In addition the `yarn.lock` file isn't added to the tarball, so it won't be available, and this `setup-env.sh` script will fail. You need to modify the `Makefile.am` file so that `yarn.lock` is added to the tarball:

```
EXTRA_DIST = \
  ...
  yarn.lock
```